Source/Data/TVADefaults.sql

```diff
@@ -86,10 +86,8 @@ GO
 
 /* LineSegment Fawg Fields */
 INSERT INTO AdditionalField (ParentTable, FieldName, ExternalDB, ExternalDBTable, ExternalDBTableKey) VALUES 
-('LineSegment', 'FromBusName','Fawg','LineSegment','FromBusName'),
-('LineSegment', 'ToBusName','Fawg','LineSegment','ToBusName'),
-('LineSegment', 'FromBus','Fawg','LineSegment','FromBus'),
-('LineSegment', 'ToBus','Fawg','LineSegment','ToBus')
+('LineSegment', 'FromBusNumber','Fawg','LineSegment','FromBusName'),
```

Comment:
This does not quite look right, But I will make that part of the other Pull Request fixing the LineSegment Wizard

Reply:
I made this change because the name is now part of the base linesegment table.